src/styles/utils/vars.scss

```diff
@@ -0,0 +1,13 @@
+$main-color: #000;
+$accent-color: #00acdc;
+$main-font-family: 'Roboto', sans-serif;
+$main-font-size: 16px;
+$logo-width: 40px;
+$logo-height: 40px;
+$header-height: 60px;
+$nav-color: #060b35;
+$card-main-color: #060b35;
+$card-hover-title-color: #34568b;
+$card-addiotioanl-color: #616070;
+$button-main-color: #00acdc;
```

Comment:
You already have a variable with this value
```suggestion
```
<img width="279" alt="image" src="https://github.com/user-attachments/assets/e0fb31b5-e736-44ef-9a22-5d64ff6aa82c">